packages/relay/tests/helpers.ts

```diff
@@ -402,6 +402,18 @@ const mockData = {
         }
     },
 
+    notSuported: {
```

Comment:
This already exist in the object and is a duplicate

Reply:
Removed.

Reply:
It's still there. I don't think you pushed your removal